src/sentry/static/sentry/app/components/events/interfaces/spans/spanTree.tsx

```diff
@@ -93,7 +95,8 @@ class SpanTree extends React.Component<PropType> {
       if (!isCurrentSpanHidden) {
         messages.push(
           <span key="spans-filtered">
-            <strong>{numOfFilteredSpansAbove}</strong> {t('spans filtered out')}
+            <strong>{numOfFilteredSpansAbove}</strong>{' '}
+            {numOfFilteredSpansAbove === 1 ? t('hidden span') : t('hidden spans')}
```

Comment:
This should use tct() to format the number into the phrase as different languages can have different sentence ordering.

Reply:
Should be fixed in https://github.com/getsentry/sentry/pull/20552/commits/28331ef95f13f89cf86e13ed8728fc1f25a72de6